tenets/codelingo/code-review-comments/avoid-renaming-imports/codelingo.yaml

```diff
@@ -0,0 +1,40 @@
+funcs:
+  - name: diffNameImport
+    type: asserter
+    body: |
+      function(a, b) {
+        var aSplit = a.toLowerCase().split("/")
+        var bSplit = b.toLowerCase().split("/")
+        return a[a.length - 1] !== b[b.length - 1]
+      }
+tenets:
+  - name: avoid-renaming-imports
+    flows:
+      codelingo/docs:
+        title: Avoid Renaming Imports
+        body: |
+          Avoid renaming imports except to avoid a name collision; good package names
+          should not require renaming. In the event of collision, prefer to rename the
+          most local or project-specific import.
+      codelingo/review:
+        comment: |
+          Avoid renaming imports except to avoid a name collision; good package names
```

Comment:
This is the tense comments should be written in. 

```
This import renames the default identifier `{{originalImportName}}` to `{{newImportName}}` but this is a bad practice. Remove `{{newImportName}}` to keep the original identifier.
```

Or maybe rename to this
```
This import is renamed to `{{badIdentName}}` but this is a bad practice. Import without specifying a new name.
```